sites/public/src/components/shared/FormSummaryDetails.tsx

```diff
@@ -187,7 +187,7 @@ const FormSummaryDetails = ({
 
   return (
     <>
-      <h3 className="form--card__sub-header">
+      <h3 className="form--card__sub-header border-none text-xl">
```

Comment:
We can return to this later, but flagging to shift to the Seeds Heading component.